src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -246,6 +246,9 @@ MulticopterAttitudeControl::Run()
 		if (_vehicle_attitude_setpoint_sub.updated()) {
 			vehicle_attitude_setpoint_s vehicle_attitude_setpoint;
 			_vehicle_attitude_setpoint_sub.update(&vehicle_attitude_setpoint);
+			const float yaw_weight = vehicle_attitude_setpoint.absolute_heading_valid ? _param_mc_yaw_weight.get() : 0.f;
```

Comment:
Looking at it now I think the controller itself should be able to handle not controlling yaw independent of the outside resetting parameters so architecturally it's not ideal to put it here. I think I'll correct that and add unit tests before we move ahead.